java/client/src/main/java/glide/api/RedisClient.java

```diff
@@ -79,8 +81,8 @@ protected RedisClient(ConnectionManager connectionManager, CommandManager comman
      * Object result = client.customCommand(new String[]{"CLIENT","LIST","TYPE", "PUBSUB"}).get();
      * </pre>
      *
-     * @param args arguments for the custom command
-     * @return a CompletableFuture with response result from Redis
+     * @param args Arguments for the custom command
+     * @return A CompletableFuture with response result from Redis
```

Comment:
As we discussed, you can simplify that - here and for all other functions (except `CreateClient` I think)
```suggestion
     * @return A response from Redis
```